django_react_proj/users-fe/src/components/UsersListRow/UsersListRow.js

```diff
@@ -0,0 +1,60 @@
+import React, {useState} from 'react';
+import axios from "axios";
+import { API_URL } from "../../constants";
+import NewUserModal from "../NewUserModal/NewUserModal";
+import ConfirmRemovalModal from "../ConfirmRemovalModal/ConfirmRemovalModal";
+import TaskButton from "../TaskButtonModal/TaskButtonModal";
+import styles from "../UserList/UserList.module.css";
+
+
+const UsersListRow = ({user,resetState}) => {
+    const [taskFlag, setTaskFlag] = useState(
+      false
+  )
+  
+  const ontoggleTaskFlag = () => {
+    setTaskFlag(previous => (
+        !previous
+      ));
+  };
+  
+  const deleteUser = async(id) => {
+    try{
+      await axios.delete(`${API_URL}${id}/`)
+      resetState()
+    } catch (error){
+      console.log(error)
+    }
+  };
+  
+  return(
+    <tr className = {taskFlag ? styles.ButtonTrue : styles.ButtonFalse} key={user.id}>
+      <td>{user.name}</td>
+      <td className={styles.Svg}>{user.password}</td>
+      <td>{user.registrationDate}</td>
+      <td align="center">
+        <TaskButton
+          id = {user.id}
+          deleteUser = {deleteUser}
+          taskFlag = {taskFlag}
+  
+          ontoggleTaskFlag = {ontoggleTaskFlag}
+        />
+        &nbsp;&nbsp;
```

Comment:
Passing custom class for each element to add a margin would be a better idea than this. Or you can use grid / flexbox for a parent

https://css-tricks.com/snippets/css/a-guide-to-flexbox/
https://css-tricks.com/snippets/css/complete-guide-grid/